### DISCUSSION TITLE
V4.3 S-Max values

### DISCUSSION THREAD

**Discussion by lb4207** [arkypita/LaserGRBL#1400] [Q&A]
I've just upgraded to V4.3 from V3.x and when I tried to run a job the the laser is very weak.
The values for S-MIN and S-MAX show 0.0% and 100.0% respectively. I entered 100, assuming that was 100%.
I've found through googling that others are using anything from 1000 to 5000 as an S-Max while some documentation says the range is 0 to 255.
Which is correct for version 4.3 of LaserGRBL?

**Comment by StuartB4** [comment]
Does'nt matter. The highest number you type in $30 will be maximum power. Whether it's 255 or 10000.
$31 should be 0, minimum power or off. 
If you set $30 to 1000 and then set S-MAX to 500 you will get half of the full power. 
If $30 is 255 then setting S-MAX to 128 will give you half of the full power.

**Reply by lb4207** [reply]
Thanks Stuart for this information.
That explains clearly how it works.
